tests/package_managers/apt_get/BUILD

```diff
@@ -76,3 +79,27 @@ set -x
 tar -xvf tests/package_managers/test_download_pkgs.tar && dpkg -i ./*.deb && apt-get install -f""",
     file = ":test_apt_get_from_tar",
 )
+
+# The gpg rules need a base image with curl and gnupg2.
+container_run_and_commit(
+    name = "test_gpg_base",
+    commands = ["apt-get update && apt-get install -y curl gnupg2"],
```

Comment:
How about we add this command to the list of add_gpg_key_from_url or else everyone has to first build an image with curl and gpg.